src/error/dfxvm.rs

```diff
@@ -46,9 +46,15 @@ pub enum Error {
 
 #[derive(Error, Debug)]
 pub enum ListError {
+    #[error("failed to fetch versions")]
```

Comment:
```suggestion
    #[error(transparent)]
```

Reply:
I followed [UpdateError](https://github.com/dfinity/dfxvm/blob/main/src/error/dfxvm.rs#L73) which also fetches the manifest URL, should we keep them consistent?

Reply:
I updated it anyway at https://github.com/dfinity/dfxvm/pull/89/commits/6bf6a71f54c29ded886d079bb1ee2d95b3756d54